src/config/renewal.hpp

```diff
@@ -5,7 +5,7 @@
 #define CONFIG_RENEWAL_HPP
 
 //quick option to disable all renewal option, used by ./configure
-//#define PRERE
```

Comment:
You could remove this and keep the RE as the default.

Reply:
oops, you're right, i'll fix it